components/tidb_query/src/rpn_expr/impl_op.rs

```diff
@@ -27,6 +27,18 @@ pub fn logical_or(arg0: &Option<i64>, arg1: &Option<i64>) -> Result<Option<i64>>
     })
 }
 
+#[rpn_fn]
+#[inline]
+pub fn logical_xor(arg0: &Option<i64>, arg1: &Option<i64>) -> Result<Option<i64>> {
+    // evaluates to 1 if an odd number of operands is nonzero, otherwise 0 is returned.
+    Ok(match (arg0, arg1) {
```

Comment:
Prefer to use the same implementation as the non-vectorized one, which is:

```
Ok(Some(((arg0 == 0) ^ (arg1 == 0)) as i64))
```

Reply:
Okay :)